sessions/biases-in-delay-distributions.qmd

```diff
@@ -29,27 +30,39 @@ library("here")
 library("cmdstanr")
 ```
 
-::: {.callout-tip}
+::: callout-tip
```

Comment:
I had no idea this works. I was going by the instructions in https://quarto.org/docs/authoring/callouts.html